guides/hack/11-built-in-types/36-enum-class-label.md

```diff
@@ -132,6 +132,10 @@ enum class F : Foo {
 // E#A === F#A is true
 ```
 
+## Enum class labels and abstract enum classes
+Abstract enum classes support labels like any other enum class. The main difference is that an abstract enum class only provides the `nameOf` static method.
+Since some of its members may be abstracted away, they do no provide the `valueOf()` or `getValues()` static methods.
```

Comment:
I recommend replacing the word 'they'. When talking of two topics, the reader might be confused about what specifically is being referred to, especially with 'its' as the previous pronoun.